src/lib/spinel/spinel.h

```diff
@@ -3055,10 +3055,13 @@ enum
     SPINEL_PROP_THREAD_NEW_DATASET = SPINEL_PROP_THREAD_EXT__BEGIN + 40,
 
     /// MAC CSL Period
-    /** Format: `S`
+    /** Format: `L`
```

Comment:
Shouldn't this trigger a `SPINEL_RCP_API_VERSION` update?

Reply:
Not really. We don't need to . This is only used in NCP (on MTD NCP) and it is not an RCP spinel property (not used by `RadioSpinel`). We need to increment `RCP_API_VERSION` when we change/add an spinel property that is used by RCP.